src/index.html

```diff
@@ -1,17 +1,24 @@
 <!DOCTYPE html>
-<html ng-app="app" lang="en">
+<html lang="en-US" ng-csp>
```

Comment:
It's a good practice to not use `eval()`, `Function(string)`, and similar functions to dynamically create and execute code from strings as well as inline resources, such as inline `<script>` and `<style>` elements.

It's true that to use the `ngHide` or `ngCloak `directive, you need to import` angular-csp.css` manually, but in this project, we don't use it, so I don't know if it makes sense to import this file out of the box.

Reply:
You are right that it is a good practice in general, but I feel it might be outside the scope of this repo (whose purpose is to demonstrate how set app a basic app).

I don't feel strongly about it, so leaving it up to you.
I do think, though, that if we keep `ngCsp` we should include `angular-csp.css` for a couple of reasons:
- It is a tiny file, so does not tangibly affect the loading of the page.
- It is not only about `ngCloak` and `ngHide`/`ngShow`. It also has styles for some kinds of animations (`.ng-anchor`, `.ng-animate-shim`).
- Even if we don't use one of the affected features directly, it is possible that one of our dependencies does.  
  (For example, the developer should not care to know if AngularJS Material relies on animations that require `.ng-anchor`/`.ng-animate-shim`.)
- Even if we don't rely on any of the features right now, it is easy to start relying in the future (either directly or indirectly via a new dependency).

Reply:
It looks like `ng-csp` was removed. Resolving this.